app/views/services/govbox/terms_api.html.erb

```diff
@@ -163,7 +163,7 @@
 
       <li><strong>Prechodné a záverečné ustanovenia</strong>
         <ol>
-          <li>Tieto Podmienky používania nadobúdajú účinnosť okamihom ich zverejnenia na webovom sídle <%= link_to 'https://ekosystem.slovensko.digital/sluzby/govbox/api-vseobecne-obchodne-podmienky' %> pokiaľ ďalej nie je uvedené inak.</li>
+          <li>Tieto Podmienky používania nadobúdajú účinnosť okamihom ich zverejnenia na webovom sídle <%= link_to "#{terms_api_services_govbox_index_url}" %> pokiaľ ďalej nie je uvedené inak.</li>
```

Comment:
Netreba interpolaciu ak tam vieš dať rovno funkciu. Taktiež _path a nie _url

Reply:
Chcem docielit aby ukazovalo celu url (ako podla dokumentu od pravnikov)
![image](https://user-images.githubusercontent.com/44023416/64920356-0edf1a00-d7b7-11e9-9fde-4a0fdbfbc5ee.png)

vtedu musim takto
`<%= link_to terms_api_services_govbox_index_url %>`
alebo takto ci?
`<%= link_to "https://ekosystem.slovensko.digital#{terms_api_services_govbox_index_path}" %>`

Reply:
@smith558 dokument od pravnikov predpoklada, ze to budes mat vytlaceny preto tam davaju celu url :D ale v klude to prve teda

Reply:
Rozumiem. Ja som sa ho snazil drzat na 100% no :D